source/administration/changelog.md

```diff
@@ -4,6 +4,133 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.14 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
+
+Release Date 2019-08-16
+
+### Compatibility
+
+#### Removed and Deprecated Features
+
+ - We are removing support for Internet Explorer (IE11) in Mattermost v5.16.0, which releases on October 16, 2019. Learn more in our [forum post](https://forum.mattermost.org/t/mattermost-is-dropping-support-for-internet-explorer-ie11-in-v5-16/7575).
+ - **Enable Plugin Health Check** removed from ``config.json``. Instead, all plugins are now automatically monitored and restarted if a plugin has crashed or is deemed unhealthy. If a plugin crashes three times within an hour, the server automatically deactivates the plugin.
+ 
+### Breaking Changes
+ - **Change related to viewing webhooks**
```

Comment:
Can we be more specific?

Reply:
@crspeller This changelog is still WIP. I'm waiting for this PR to be reviewed: https://github.com/mattermost/docs/pull/2864.

Reply:
I've commented on the linked PR.